src/server/journal/journal_slice.cc

```diff
@@ -111,9 +114,11 @@ error_code JournalSlice::Close() {
 
   return ec;
 }
+#endif
 
 void JournalSlice::AddLogRecord(const Entry& entry, bool await) {
   DCHECK(ring_buffer_);
+#if 0
```

Comment:
I believe we need the lsn++ operation inside the commented out code.
Did you run the replication regression tests and they passed?

Reply:
fixed